schema/rpc.fbs

```diff
@@ -201,7 +201,8 @@ table TapDetectionSettings {
     mounting_reset_delay: float = null;
     mounting_reset_enabled: bool = null;
     mounting_reset_taps: uint8 = null;
-    /// Iff true, disables reset behavior of tap detection and sends a
+    number_trackers_over_threshold: uint8 = null;
```

Comment:
Adding a field in the middle of a table like this, breaks all subsequent fields. This is because the names of fields dissapear and are only referenced based on position, so all the positions of subsequent fields are off by one.

Please move it to the end of the table, as adding to the table's end is not a breaking change.

Reply:
I somehow forgot about this thanks!